src/EditorFeatures/Core.Wpf/Adornments/AbstractAdornmentManagerProvider.cs

```diff
@@ -18,9 +18,9 @@ internal abstract class AbstractAdornmentManagerProvider<TTag> :
         IWpfTextViewCreationListener
         where TTag : GraphicsTag
     {
-        private readonly IThreadingContext _threadingContext;
-        private readonly IViewTagAggregatorFactoryService _tagAggregatorFactoryService;
-        private readonly IAsynchronousOperationListener _asyncListener;
+        protected readonly IThreadingContext _threadingContext;
+        protected readonly IViewTagAggregatorFactoryService _tagAggregatorFactoryService;
+        protected readonly IAsynchronousOperationListener _asyncListener;
```

Comment:
protected members should not use `_` names.  Use PascalCasing in this case.

Reply:
fixed